docs/evm-sidechain/witness-servers.md

```diff
@@ -0,0 +1,16 @@
+---
+html: witness-server.html
+blurb: Witness servers listen for bridge transactions and provide attestations.
+labels:
+  - Interoperability
+status: not_enabled
+---
+# Witness Servers
+
+A witness server enables bridge operators to listen for transactions on the connected chains and sign attestations. Attestations are used as proof that events on a chain occurred.
```

Comment:
I might put _attestations_ in italics, since it's defined in the following sentence.